find_new/src/find_new.py

```diff
@@ -65,26 +81,23 @@ def submit_jobs_for_granule(hyp3, granule, event_id):
     prepared_jobs.append(hyp3.prepare_rtc_job(granule=granule['granuleName']))
     granule_lists.append([granule])
 
-    neighbors = asf_search.get_nearest_neighbors(granule['granuleName'])
+    try:
+        neighbors = asf_search.get_nearest_neighbors(granule['granuleName'])
+    except requests.HTTPError as e:
+        if e.response.status_code in range(400, 500):
+            raise GranuleError()
+        if e.response.status_code >= 500:
+            print(f'unable to find pairs for {granule["granuleName"]} skipping...')
```

Comment:
should we log `str(e)` so we know the nature of the failure?
`Unable to find InSAR pairs for {}: {str(e)}`?